src/js/index.js

```diff
@@ -1,4 +1,6 @@
 import reset from "/src/css/reset.css?inline";
 import { Elm } from "/src/Main.elm";
 
-const app = Elm.Main.init({});
+import guides from "../../data/guides.json";
```

Comment:
This file is missing - prevents the site from running locally

Reply:
before first run you need to do `npm run generate`

Reply:
ahh! Thanks

Reply:
this tripped me up as well when it first happened

Reply:
Yeah - we could add it to happen automatically for dev but I tend to not because it doesn't need to happen unless the content changes so it feels like an extra overhead - though quick and trivial. Just makes the process seem unnecessarily heavy to me.

Mainly a psychological thing though - so feel free to add in if others think we should.